tests/transform/integration/test_es_integration.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+import subprocess
+import os
+
+class MyTestCase(unittest.TestCase):
+
+    
+    def setUp(self):
+        current_path = os.path.dirname(__file__)
+        root_dir = os.path.normpath(os.path.join(current_path, "../../../"))
+        result = subprocess.run([root_dir + "/main.py", "trans", "es", root_dir + "/tests/transform/integration/fixtures/VIB.json", "--data-dir", root_dir + "/tests/transform/integration/fixtures/brapi_pheno_source"])
```

Comment:
The generated files should be `diff` with the fixtures for the test to be relevant, right?

Reply:
Yes, done in each test_all